tests/components/airzone_cloud/test_climate.py

```diff
@@ -74,6 +74,25 @@ async def test_airzone_create_climates(hass: HomeAssistant) -> None:
     assert state.attributes.get(ATTR_TARGET_TEMP_STEP) == API_TEMPERATURE_STEP
     assert state.attributes.get(ATTR_TEMPERATURE) == 24.0
 
+    # Installations
+    state = hass.states.get("climate.house")
+    assert state.state == HVACMode.COOL
+    assert state.attributes.get(ATTR_CURRENT_HUMIDITY) == 27
+    assert state.attributes.get(ATTR_CURRENT_TEMPERATURE) == 22.0
+    assert state.attributes.get(ATTR_HVAC_ACTION) == HVACAction.COOLING
+    assert state.attributes.get(ATTR_HVAC_MODES) == [
+        HVACMode.HEAT_COOL,
+        HVACMode.COOL,
+        HVACMode.HEAT,
+        HVACMode.FAN_ONLY,
+        HVACMode.DRY,
+        HVACMode.OFF,
+    ]
+    assert state.attributes.get(ATTR_MAX_TEMP) == 30
+    assert state.attributes.get(ATTR_MIN_TEMP) == 15
+    assert state.attributes.get(ATTR_TARGET_TEMP_STEP) == API_TEMPERATURE_STEP
+    assert state.attributes.get(ATTR_TEMPERATURE) == 23.3
```

Comment:
Please use `[]` here as the key should not be missing

Reply:
Done 76940724d61cfd07afc5f515f7d90d91acbec322